test/utils/isSameDay_spec.js

```diff
@@ -15,6 +15,15 @@ describe('isSameDay', () => {
     expect(isSameDay(today, tomorrow)).to.equal(false);
   });
 
+  it('returns false for same days of week', () => {
+    // Flags accidentally use of moment's day() function, which returns index
```

Comment:
Does this mean that the function had a bug previously? Or does it mean that "is same day" isn't the right semantic, and we want "is same date"? Are we sure that every use of `isSameDay` wanted "same date" and not "same day"?

Either way, if we want "same date" semantics, we need to rename the function, since "same day" means "same day of week" in JS date parlance.